2021-10-06 15:53:26,309 INFO    MainThread:24974 [wandb_setup.py:_flush():69] setting env: {}
2021-10-06 15:53:26,309 INFO    MainThread:24974 [wandb_setup.py:_flush():69] setting login settings: {'anonymous': 'must'}
2021-10-06 15:53:26,310 INFO    MainThread:24974 [wandb_init.py:_log_setup():342] Logging user logs to /home/inge/Pytorch-UNet/wandb/run-20211006_155326-hwkw33yl/logs/debug.log
2021-10-06 15:53:26,310 INFO    MainThread:24974 [wandb_init.py:_log_setup():343] Logging internal logs to /home/inge/Pytorch-UNet/wandb/run-20211006_155326-hwkw33yl/logs/debug-internal.log
2021-10-06 15:53:26,310 INFO    MainThread:24974 [wandb_init.py:init():375] calling init triggers
2021-10-06 15:53:26,310 INFO    MainThread:24974 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-06 15:53:26,310 INFO    MainThread:24974 [wandb_init.py:init():424] starting backend
2021-10-06 15:53:26,310 INFO    MainThread:24974 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-06 15:53:26,322 INFO    MainThread:24974 [backend.py:ensure_launched():135] starting backend process...
2021-10-06 15:53:26,326 INFO    MainThread:24974 [backend.py:ensure_launched():139] started backend process with pid: 24982
2021-10-06 15:53:26,327 INFO    MainThread:24974 [wandb_init.py:init():429] backend started and connected
2021-10-06 15:53:26,328 INFO    MainThread:24974 [wandb_init.py:init():477] updated telemetry
2021-10-06 15:53:26,329 INFO    MainThread:24974 [wandb_init.py:init():500] communicating current version
2021-10-06 15:53:26,876 INFO    MainThread:24974 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-06 15:53:26,876 INFO    MainThread:24974 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-06 15:53:27,239 INFO    MainThread:24974 [wandb_init.py:init():540] starting run threads in backend
2021-10-06 15:53:29,333 INFO    MainThread:24974 [wandb_run.py:_console_start():1601] atexit reg
2021-10-06 15:53:29,334 INFO    MainThread:24974 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-10-06 15:53:29,335 INFO    MainThread:24974 [wandb_run.py:_redirect():1480] Redirecting console.
2021-10-06 15:53:29,337 INFO    MainThread:24974 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-06 15:53:29,337 INFO    MainThread:24974 [wandb_init.py:init():565] run started, returning control to user process
2021-10-06 15:53:29,338 INFO    MainThread:24974 [wandb_run.py:_config_callback():843] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': False}
2021-10-06 15:53:29,380 INFO    MainThread:24974 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-10-06 15:53:29,380 INFO    MainThread:24974 [wandb_run.py:_restore():1543] restore
2021-10-06 15:53:31,864 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-06 15:53:32,152 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-06 15:53:32,279 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 4550
}

2021-10-06 15:53:32,382 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 8236
}

2021-10-06 15:53:32,484 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 8236
}

2021-10-06 15:53:32,587 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8236
  total_bytes: 8236
}

2021-10-06 15:53:32,689 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8236
  total_bytes: 8236
}

2021-10-06 15:53:32,792 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8236
  total_bytes: 8236
}

2021-10-06 15:53:32,894 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8236
  total_bytes: 8236
}

2021-10-06 15:53:32,996 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8236
  total_bytes: 8236
}

2021-10-06 15:53:33,311 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8236
  total_bytes: 8236
}

2021-10-06 15:53:33,414 INFO    MainThread:24974 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8236
  total_bytes: 8236
}

2021-10-06 15:53:34,509 INFO    MainThread:24974 [wandb_run.py:_show_files():1915] logging synced files
